Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.9.16/x64/lib/python3.9/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/opt/hostedtoolcache/Python/3.9.16/x64/lib/python3.9/site-packages/nbclient/client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/opt/hostedtoolcache/Python/3.9.16/x64/lib/python3.9/site-packages/nbclient/util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/opt/hostedtoolcache/Python/3.9.16/x64/lib/python3.9/site-packages/nbclient/util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "/opt/hostedtoolcache/Python/3.9.16/x64/lib/python3.9/asyncio/base_events.py", line 647, in run_until_complete
    return future.result()
  File "/opt/hostedtoolcache/Python/3.9.16/x64/lib/python3.9/site-packages/nbclient/client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "/opt/hostedtoolcache/Python/3.9.16/x64/lib/python3.9/site-packages/nbclient/client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/opt/hostedtoolcache/Python/3.9.16/x64/lib/python3.9/site-packages/nbclient/client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
viewer = py3Dmol.view()
viewer.addModel(phase.to_cif_str(),'cif',{'doAssembly':True,'duplicateAssemblyAtoms':True,'normalizeAssembly':True})
viewer.setStyle({'sphere':{'colorscheme':'Jmol','scale':.2},'stick':{'colorscheme':'Jmol', 'radius': 0.1}})
viewer.addUnitCell()
viewer.replicateUnitCell(2,2,2)
viewer.zoomTo()
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mAttributeError[0m                            Traceback (most recent call last)
Cell [0;32mIn[6], line 2[0m
[1;32m      1[0m viewer [38;5;241m=[39m py3Dmol[38;5;241m.[39mview()
[0;32m----> 2[0m viewer[38;5;241m.[39maddModel([43mphase[49m[38;5;241;43m.[39;49m[43mto_cif_str[49m(),[38;5;124m'[39m[38;5;124mcif[39m[38;5;124m'[39m,{[38;5;124m'[39m[38;5;124mdoAssembly[39m[38;5;124m'[39m:[38;5;28;01mTrue[39;00m,[38;5;124m'[39m[38;5;124mduplicateAssemblyAtoms[39m[38;5;124m'[39m:[38;5;28;01mTrue[39;00m,[38;5;124m'[39m[38;5;124mnormalizeAssembly[39m[38;5;124m'[39m:[38;5;28;01mTrue[39;00m})
[1;32m      3[0m viewer[38;5;241m.[39msetStyle({[38;5;124m'[39m[38;5;124msphere[39m[38;5;124m'[39m:{[38;5;124m'[39m[38;5;124mcolorscheme[39m[38;5;124m'[39m:[38;5;124m'[39m[38;5;124mJmol[39m[38;5;124m'[39m,[38;5;124m'[39m[38;5;124mscale[39m[38;5;124m'[39m:[38;5;241m.2[39m},[38;5;124m'[39m[38;5;124mstick[39m[38;5;124m'[39m:{[38;5;124m'[39m[38;5;124mcolorscheme[39m[38;5;124m'[39m:[38;5;124m'[39m[38;5;124mJmol[39m[38;5;124m'[39m, [38;5;124m'[39m[38;5;124mradius[39m[38;5;124m'[39m: [38;5;241m0.1[39m}})
[1;32m      4[0m viewer[38;5;241m.[39maddUnitCell()

[0;31mAttributeError[0m: 'Phase' object has no attribute 'to_cif_str'
AttributeError: 'Phase' object has no attribute 'to_cif_str'

